### ISSUE TITLE
silence speech and play sound

### ISSUE THREAD

**Issue by joshknnd1982** [CyrilleB79/outlookExtended#18]
Can you add a feature, so when a checkbox is checked, when I press enter to open an email in outlook, NVDA does not read the first line of the message, instead it plays a sound when the message is loaded and is ready to be reviewed? thanks.
